src/highgui.rs

```diff
@@ -118,19 +139,41 @@ pub enum MouseEventType {
     MouseHWheel = 11,
 }
 
+impl From<c_int> for MouseEventType {
```

Comment:
Do you mind reference `num_derive` and detive `FromPrimitive` trait instead?

Reply:
One potential problem is that `c_int` could be an i32 or an i64. I couldn't think of another simple way to solve that issue and settled for this for now.